samples/drivers/eeprom/sample.yaml

```diff
@@ -2,20 +2,29 @@ sample:
   name: EEPROM Sample
 common:
     tags: eeprom
-    harness: console
     harness_config:
       type: one_line
       regex:
         - "Reset the MCU to see the increasing boot counter."
```

Comment:
Now the common `harness_config` does not apply to each test case, as one of them is now using `harness: keyboard`.